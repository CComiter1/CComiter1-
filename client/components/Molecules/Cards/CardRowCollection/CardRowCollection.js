import React from 'react'
import styled from 'styled-components'

import {
  CardRow,
} from '../../../Molecules'

const Wrapper = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: space-between;
`

export default props => (
  <Wrapper>
    {
      props.information.map(info => (
        <CardRow key={info.label} label={info.label}>{info.data}</CardRow>
      ))
    }
  </Wrapper>
)
