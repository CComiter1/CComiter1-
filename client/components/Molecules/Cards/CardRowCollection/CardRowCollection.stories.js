import React from 'react'
import { storiesOf } from '@storybook/react'

import CardRowCollection from './CardRowCollection'

storiesOf('Molecules/Cards', module)
  .add('CardRowCollection', () => (
    <CardRowCollection
      information={[
        { label: 'Customer ID#', data: '' },
        { label: 'Name', data: 'Asim' },
        { label: 'Account Status', data: 'good' },
        { label: 'Last Pass. Reset', data: 'today' },
      ]}
    />
  ))
