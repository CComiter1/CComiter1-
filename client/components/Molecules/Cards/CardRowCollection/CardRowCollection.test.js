import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import CardRowCollection from './CardRowCollection'

describe('CardRowCollection', () => {
  it('renders correctly', () => {
    const information = [
      { label: 'Customer ID#', data: '' },
      { label: 'Name', data: 'Asim' },
      { label: 'Account Status', data: 'good' },
      { label: 'Last Pass. Reset', data: 'today' },
    ]
    const component = renderer.create(
      <CardRowCollection information={information} />
    ).toJSON()
    expect(component).toMatchSnapshot()
  })
})
