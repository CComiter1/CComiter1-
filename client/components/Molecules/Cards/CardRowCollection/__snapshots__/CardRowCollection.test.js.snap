// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardRowCollection renders correctly 1`] = `
.c3 {
  font-family: 'Roboto',sans-serif;
  margin: 0;
}

.c2 {
  font-family: 'Roboto',sans-serif;
  margin: 0;
  font-weight: bold;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  width: 100%;
  padding: 20px 20px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: space-between;
  -webkit-box-align: space-between;
  -ms-flex-align: space-between;
  align-items: space-between;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <p
      className="c2"
    >
      Customer ID#
      :
    </p>
    <p
      className="c3"
    >
      
    </p>
  </div>
  <div
    className="c1"
  >
    <p
      className="c2"
    >
      Name
      :
    </p>
    <p
      className="c3"
    >
      Asim
    </p>
  </div>
  <div
    className="c1"
  >
    <p
      className="c2"
    >
      Account Status
      :
    </p>
    <p
      className="c3"
    >
      good
    </p>
  </div>
  <div
    className="c1"
  >
    <p
      className="c2"
    >
      Last Pass. Reset
      :
    </p>
    <p
      className="c3"
    >
      today
    </p>
  </div>
</div>
`;
